project/NativeImage.scala

```diff
@@ -86,22 +86,29 @@ object NativeImage {
     initializeAtRuntime: Seq[String]         = Seq.empty,
     initializeAtBuildtime: Seq[String]       = defaultBuildTimeInitClasses,
     mainClass: Option[String]                = None,
-    additionalCp: Seq[String]                = Seq(),
+    additionalCp: () => Seq[String]          = () => Seq(),
     verbose: Boolean                         = false,
     includeRuntime: Boolean                  = true
   ): Def.Initialize[Task[Unit]] = Def
     .task {
-      val log            = state.value.log
-      val javaHome       = System.getProperty("java.home")
-      val subProjectRoot = baseDirectory.value
-      val nativeImagePath =
+      val log = state.value.log
+
+      def nativeImagePath(path: String): String =
         if (Platform.isWindows)
-          s"$javaHome\\bin\\native-image.cmd"
-        else s"$javaHome/bin/native-image"
+          s"$path\\bin\\native-image.cmd"
+        else s"$path/bin/native-image"
+
+      val localJdk = baseDirectory.value / ".." / ".." / "jdk"
```

Comment:
This is supposed to pick up the minimal JDK downloaded with `jdk.sh`, right? Would be nic to embed the functionality of `jdk.sh` into sbt.

Reply:
sbt uses the script to build the JDK since d29c9a1 - we build the `runner` executable only on Unix, so it may be sufficient to keep it this way for now.